### DISCUSSION TITLE
blockchain/blockchain/week3

### DISCUSSION THREAD

**Discussion by giscus** [wuyuchuan617/docusaurus-2#8] [Announcements]
# blockchain/blockchain/week3

Solidity、ERC-20

https://anytype.vercel.app/blockchain/blockchain/week3

**Comment by 42bchen** [comment]
1. `Bank.sol:L82` 這邊的 approve 好像怪怪的
2. private 跟 internal 變數可以給他前面加底線，ex: `mapping (address => uint) private Accounts`
3. 我是跪著看你的作業的，你的作業太帥惹吧 🫡
